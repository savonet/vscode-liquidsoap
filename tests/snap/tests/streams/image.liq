#!../../liquidsoap ../test.liq

s = (single("file1.png"):source(video=canvas))
output.dummy(fallible=true, s)

thread.run(delay=3., test.pass)
