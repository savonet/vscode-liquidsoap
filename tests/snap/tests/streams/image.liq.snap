>#!../../liquidsoap ../test.liq
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap comment.line.number-sign.liquidsoap
>
>s = (single("file1.png"):source(video=canvas))
#^^^^^^^^^^^^ source.liquidsoap
#            ^ source.liquidsoap string.quoted.double.liquidsoap
#             ^^^^^^^^^ source.liquidsoap string.quoted.double.liquidsoap
#                      ^ source.liquidsoap string.quoted.double.liquidsoap
#                       ^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap
>output.dummy(fallible=true, s)
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.liquidsoap
>
>thread.run(delay=3., test.pass)
#^^^^^^^^^^^^^^^^^ source.liquidsoap
#                 ^ source.liquidsoap constant.numeric.integer
#                  ^^^^^^^^^^^^^^ source.liquidsoap
>